emulators/blink/Portfile

```diff
@@ -10,7 +10,7 @@ legacysupport.newest_darwin_requires_legacy 15
 github.setup        jart blink 1.1.0
 # Change github.tarball_from to 'releases' or 'archive' next update
 github.tarball_from tarball
```

Comment:
Can we try changing this to `archive` while we're at it? Checksums will need to be updated.